java/code/src/com/redhat/rhn/manager/audit/AuditManager.java

```diff
@@ -36,15 +36,18 @@
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 /**
  * AuditManager
  */
-public class AuditManager /* extends BaseManager */ {
+public class AuditManager {
 
-    private static Logger log = LogManager.getLogger(AuditManager.class);
+    private static final Logger LOGGER = LogManager.getLogger(AuditManager.class);
+    private static Map<String, String[]> auditTypeMappings = null;
```

Comment:
> Would you mind explaining the reason for defining it as a class level? 

getAuditTypeMap creates the same map each time it's called, so I figured it would be a quick win to cache the map in a class level field.

>  it is generally recommended to avoid initializing objects with null

Totally agree!